docs/tools/datadoc.md

```diff
@@ -382,15 +382,15 @@ The following formats are currently available:
     This will show the same output as in Ex 4.
 
 
-### load
-Loads documented dataset from a storage.
-Running `datadoc load --help` will show the following help message:
+### fetch
+Fetch documented dataset from a storage.
+Running `datadoc fetch --help` will show the following help message:
 
 ```
-usage: datadoc load [-h] [--output FILENAME] iri
+usage: datadoc fetch [-h] [--output FILENAME] iri
 
 positional arguments:
-  iri                   IRI of dataset to load.
+  iri                   IRI of dataset to fetch.
```

Comment:
If I have understood correctly it does not fetch the dataset, but fetches the documentation of that dataset.

Reply:
No, it fetch the actual dataset. The `find` sub-command is used to get the data documentation with the datadoc tool